app/helpers/rademade_admin/field_helper.rb

```diff
@@ -19,6 +19,8 @@ def display_item_value(item, data_item)
       end
     elsif data_item.has_uploader?
       RademadeAdmin::Upload::PreviewService.new(value).uploaded_file_html
+    elsif data_item.boolean_field?
+      value ? '✔' : '×'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.